### PR TITLE
Contribution

### PR DESCRIPTION
**Pull request by carmllab**
# Change

Updating routeTable with providing routes as output
Updated the readme

[Network: RouteTables](https://github.com/carmllab/ResourceModules/actions/workflows/ms.network.routetables.yml)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

## Checklist

- [ ] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

### CODE REVIEW

File: arm/Microsoft.Insights/components/version.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
     "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-    "version": "0.4"
+    "version": "0.1"
```
Comment (MrMCake) [inline arm/Microsoft.Insights/components/version.json:3]: I guess undo?

File: arm/Microsoft.OperationalInsights/workspaces/version.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
     "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-    "version": "0.4"
+    "version": "0.1"
```
Comment (MrMCake) [inline arm/Microsoft.OperationalInsights/workspaces/version.json:3]:
```suggestion
    "version": "0.4"
```

File: arm/Microsoft.Storage/storageAccounts/version.json [hunk 3]
```diff
@@ -1,4 +1,4 @@
 {
     "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-    "version": "0.4"
+    "version": "0.1"
```
Comment (MrMCake) [inline arm/Microsoft.Storage/storageAccounts/version.json:3]:
```suggestion
    "version": "0.4"
```

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**